Assets/Source/Components/Actor/ActorDashBehavior.cs

```diff
@@ -126,6 +138,10 @@ private Vector2 DashLeft(Vector2 externalVelocity)
                     return EndDash(externalVelocity);
                 }
 
+                ParticleSystem.ShapeModule shapeModule = dashParticles.shape;
+                shapeModule.rotation = shapeModule.rotation.Copy(y: 90.0f);
```

Comment:
Also yes this looks gross but its the only way to modify the weird struct design they did for this. You can look here an the accepted answer for a good explanation: https://stackoverflow.com/questions/40130332/understanding-the-struct-in-unity-particlesystem

Reply:
no big deal, my code does the same stuff for the asteroid explosions and stuff